error-handler.js

```diff
@@ -258,9 +258,17 @@ createHandler = function createHandler(options) {
         }
       },
 
+      shouldContinue = function
```

Comment:
OK. This changes the previous behavior of the library. I'm a new user, with this interest.
Do you have any experience with 503 Too Busy errors as they relate to express? I actually don't, so I'm more than willing to go along with this interpretation. Makes things a lot easier, and doesn't effect my use case.